File: iOS/Component/TestPods_OC/TestPods_OC/ViewController.m
//
//  ViewController.m
//  TestPods_OC
//
//  Created by 启业云03 on 2021/3/23.
//

#import "ViewController.h"

//#import <OnlyOCDemo/AllHeader.h>
#import <OnlyOCDemo/FunctionB.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.

//    [FunctionA printSomethingA:@"A打印"];
    
    [FunctionB printSomethingB:@"B打印"];
}


@end

File: iOS/SDK/Projects/Swift_Test/Swift_Test/ViewController.swift
//
//  ViewController.swift
//  Swift_Test
//
//  Created by 启业云03 on 2021/8/25.
//

import UIKit
import LXScan;

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        let scanVC = LXScan.QYYScanViewController()
        self .present(scanVC, animated: true, completion: nil)
        
    }

}

